<launch>
    <remap from="joy" to="joy_remote" />
    <node pkg="joy" type="joy_node" name="joy_node"/>
</launch>